lib/Config.php

```diff
@@ -173,7 +173,7 @@ public static function getWebsiteConfig($language = null)
             $siteId = null;
             if (Model\Site::isSiteRequest()) {
                 $siteId = Model\Site::getCurrentSite()->getId();
-            } elseif (Tool::isFrontentRequestByAdmin()) {
+            } elseif (Tool::isFrontendRequestByAdmin()) {
```

Comment:
What has changed here? Can you maybe revert those changes?

Reply:
isFrontentRequestByAdmin is deprecated:
https://github.com/pimcore/pimcore/blob/7bff13be32bf5962a323ac56549c1a6b2ab12a73/lib/Tool.php#L362-L372

Should we write a deprecated warning if someone uses it and remove it in 6.0?

Reply:
Ah, sorry, didn't see the difference at the first sight :) 
It's fine as it is, thanks!